Introduction to Coroutines and Channels/06_StructuredConcurrency.md

```diff
@@ -1,12 +1,12 @@
 # Structured concurrency
  
 _Coroutine scope_ is responsible for the structure and parent-child relationships between different coroutines.
-You always start new coroutines inside a scope.
+New coroutines always start inside a scope.
 _Coroutine context_ stores additional technical information used to run a given coroutine,
 like the dispatcher specifying the thread or threads the coroutine should be scheduled on.
 
-When `launch`, `async`, or `runBlocking` start a new coroutine, they automatically create the corresponding scope.
-All these functions take lambda with receiver as an argument, and the type of the implicit receiver is `CoroutineScope`:
+When `launch`, `async`, or `runBlocking` are used to start a new coroutine, they automatically create the corresponding scope.
+All these functions take a lambda with the receiver as an argument, and the implicit reciever type is the `CoroutineScope`:
```

Comment:
'lambda with receiver' is the feature name (https://kotlinlang.org/docs/reference/lambdas.html#function-literals-with-receiver)